script/common/deployment/01_deployToken.s.sol

```diff
@@ -31,13 +36,29 @@ contract DeployToken is Script {
         // deploy Token facet
         Token tokenFacet = new Token();
 
+        uint256 canDeployTokenBridgeFacet = vm.envUint(
+            "CAN_DEPLOY_TOKEN_BRIDGE_FACET"
+        );
+
+        ITokenBridge tokenBridgeFacet;
+
+        if (canDeployTokenBridgeFacet == 1) {
+            // deploy TokenBridge facet
+            tokenBridgeFacet = new TokenBridge(gateway, gasService);
+
+            console.log(
+                "TokenBridge Facet Address: ",
+                address(tokenBridgeFacet)
+            );
+        }
+
         // deploy TokenProxy
         TokenProxy tokenProxy = new TokenProxy(name, symbol);
 
         console.log("Token Facet Address: ", address(tokenFacet));
         console.log("Token Proxy Address: ", address(tokenProxy));
 
-        writeTokenProxyAddress(address(tokenProxy));
+        // writeTokenProxyAddress(address(tokenProxy));
```

Comment:
Since you asked me to write also the script for cutting the bridge facet I imagined you would have needed a scenario where the bridge facet could have been not deployed at immediately. In any case I'll modify it